kpi/utils/usage_calculator.py

```diff
@@ -278,3 +278,78 @@ def _get_cache_hash(self):
             return f'user-{self.user.id}'
         else:
             return f'organization-{self.organization.id}'
+
+
+class BillingAndUsageCalculator:
```

Comment:
Please organize the file so that classes and methods are listed alphabetically (e.g. BillingAndUsageCalculator >  ServiceUsageCalculator, calculate_usage_batch  > get_effective_user_id,  just touch the code you added).